config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Refinery::Core::Engine.routes.draw do
   namespace :blog, :path => Refinery::Blog.page_url do
     root :to => "posts#index"
-    resources :posts, :only => [:show]
+    resources :posts, path: '', :only => [:show] do
```

Comment:
That's a breaking change. We should do a redirect from `blog/posts/1` to `blog/1`.